aiqleads/utils/logging.py

```
@@ -2,17 +1,97 @@
+        # File handler (log to data/logs directory)
+        try:
+            log_dir = os.path.join("aiqleads", "data", "logs")
+            os.makedirs(log_dir, exist_ok=True)
+            
+            # Set up rotating file handler
+            log_file = os.path.join(log_dir, "aiqleads.log")
+            file_handler = logging.handlers.RotatingFileHandler(
+                log_file,
+                maxBytes=5*1024*1024,  # 5MB
+                backupCount=5,
+                encoding='utf-8'
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)